source_me.sh

```diff
@@ -5,5 +5,12 @@ export C_INCLUDE_PATH=`pwd`/include:$C_INCLUDE_PATH
 export CPLUS_INCLUDE_PATH=`pwd`/include:$CPLUS_INCLUDE_PATH
 export INCLUDE_PATH=`pwd`/include:$INCLUDE_PATH
 export PATH=`pwd`/bin:$PATH
-export CC=$(which gcc)
-export CXX=$(which g++)
+if [[ "x$CC" == "x" ]]
```

Comment:
Stupid question, but why not just `[ -z "$CC" ] ` (assuming you're checking if it's unset _or_ empty) or, if you _really_ don't want to clobber it even if it's set to the empty string (although I don't think that's needed here), `[ -z ${CC+x} ]`?